File: app/src/main/java/com/sparsh/todoapp/DetailActivity.java
package com.sparsh.todoapp;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.TextView;

public class DetailActivity extends AppCompatActivity {
    TextView textViewTitle,textViewDescription;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_detail);
        bindViews();
        setupIntentData();
    }

    private void setupIntentData() {
        Intent intent = getIntent();
        String title = intent.getStringExtra("title");
        String description = intent.getStringExtra("description");
        textViewTitle.setText(title);
        textViewDescription.setText(description);

    }

    private void bindViews() {
        textViewTitle = findViewById(R.id.textViewTitle);
        textViewDescription = findViewById(R.id.textViewDescription);
    }
}

File: app/src/main/java/com/sparsh/todoapp/views/DetailActivity.kt
package com.sparsh.todoapp.views

import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import com.sparsh.todoapp.R

class DetailActivity:AppCompatActivity() {
    val TAG = "DetailActivity"
    lateinit var  textViewTitle:TextView
    lateinit var textViewDescription:TextView
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_detail)
        bindViews()
        setIntentData()
    }

    private fun bindViews() {
        textViewTitle = findViewById(R.id.textViewTitle)
        textViewDescription = findViewById(R.id.textViewDescription)


    }

    private fun setIntentData() {
        val intent = intent
        var title = intent.getStringExtra("title")
        var description = intent.getStringExtra("description")
        textViewTitle.text = title
        textViewDescription.text = description
    }

}